lib/report.c

```diff
@@ -87,6 +87,9 @@ grn_report_table(grn_ctx *ctx,
       GRN_TEXT_PUT(ctx, &description, name, name_size);
       GRN_TEXT_PUTS(ctx, &description, ">");
     }
+    if (target != table) {
+      grn_obj_unref(ctx, target);
```

Comment:
This is invalid.
We use `target` after this `grn_obj_unref` for `target = grn_ctx_at(ctx, target->header.domain)`. If `target` is closed by this `grn_obj_unref()`, `target->header.domain` touches invalid memory.

Reply:
Right, I have fixed it.